build-system/common/check-package-manager.js

```diff
@@ -183,9 +183,7 @@ function getNodeLatestLtsVersion(distributionsJson) {
 // If yarn is being run, perform a version check and proceed with the install.
 function checkYarnVersion() {
   const yarnVersion = getStdout(yarnExecutable + ' --version').trim();
```

Comment:
Can we add a TODO here to revert #30478 when a new version of yarn is published?

Reply:
Makes sense, adding now.